d3rlpy/base.py

```diff
@@ -356,6 +366,77 @@ def fit(
             scorers: list of scorer functions used with `eval_episodes`.
             shuffle: flag to shuffle transitions on each epoch.
 
+        Returns:
+            results: list of result tuples (epoch, metrics) per epoch.
+
+        """
+        results = list(
+            self.fitter(
+                episodes,
+                n_epochs,
+                save_metrics,
+                experiment_name,
+                with_timestamp,
+                logdir,
+                verbose,
+                show_progress,
+                tensorboard,
+                eval_episodes,
+                save_interval,
+                scorers,
+                shuffle,
+            )
+        )
+        return results
+
+    def fitter(
+        self,
+        episodes: List[Episode],
+        n_epochs: int = 1000,
+        save_metrics: bool = True,
+        experiment_name: Optional[str] = None,
+        with_timestamp: bool = True,
+        logdir: str = "d3rlpy_logs",
+        verbose: bool = True,
+        show_progress: bool = True,
+        tensorboard: bool = True,
+        eval_episodes: Optional[List[Episode]] = None,
+        save_interval: int = 1,
+        scorers: Optional[Dict[str, Callable[[Any, List[Episode]], float]]] = None,
+        shuffle: bool = True,
+    ) -> Generator:
```

Comment:
Please change this as follows.
```
-> Generator[Tuple[int, Dict[str, float]], None, None]
```

Reply:
Nice I tried to write the right output but then received assertion typing errors,I guess I needed the last two None,None